packages/kit/src/packaging/test/fixtures/typescript/svelte.config.cjs

```diff
@@ -1,5 +1,10 @@
 const preprocess = require('svelte-preprocess');
 
 module.exports = {
-	preprocess: preprocess()
+	preprocess: preprocess(),
+	kit: {
+		package: {
+			override: (pkg) => ({ ...pkg, svelte: 'index.js' })
```

Comment:
I think `svelte` is being added automatically now, right? So this test seems a bit confusing that we're adding something that is normally already there (or maybe it's updating it?). Perhaps it'd be clearer to just add something `foo: 'bar'`?

Reply:
Yes, this test is based on the issue, which needed to replace the `svelte` key (`src/lib/index.ts` -> `index.js`)